js-rattler/Cargo.toml

```diff
@@ -31,14 +32,29 @@ wee_alloc = { version = "0.4.5", optional = true }
 thiserror = "2.0.3"
 
 rattler_conda_types = { path = "../crates/rattler_conda_types" }
+rattler_repodata_gateway = { path = "../crates/rattler_repodata_gateway", features = ["gateway"] }
+rattler_solve = { path = "../crates/rattler_solve", default-features = false, features = ["resolvo"] }
+
+# By adding the `libbz2-rs-sys` feature we ensure that bzip2 is using the rust
+# implementation. This is important because the C implementation is not
+# compatible with wasm.
+bzip2 = { version = "0.5.1", features = ["libbz2-rs-sys"] }
 
 [dev-dependencies]
 wasm-bindgen-test = "0.3.45"
 
 [profile.release]
-# Tell `rustc` to optimize for small code size.
+## Tell `rustc` to optimize for small code size.
 opt-level = "s"
 lto = true
 
+[patch.crates-io]
+# We want to use version 0.4.19 but its not released yet.
+# https://github.com/Nullus157/async-compression/pull/319
+async-compression = { git = "https://github.com/Nullus157/async-compression.git" }
+bzip2 = { git = "https://github.com/baszalmstra/bzip2-rs.git", branch = "fix/wasm" }
```

Comment:
I need to upstream this before merging.

Reply:
See: https://github.com/trifectatechfoundation/bzip2-rs/pull/131

Reply:
Done, right?!

Reply:
Yeah, now all we need is an async compression release! But we can merge without it

Reply:
Lets go! 👍